### PR TITLE
Tc02_User can't login with blank Username textbox

### PR DESCRIPTION
**Pull request by taile47711**

### CODE REVIEW

File: src/test/java/tests/LoginTest.java [hunk 1]
```diff
@@ -23,4 +25,21 @@ public void tc001_UserCanLogIntoRailwayWithValidUsernameAndPassword() {
...
+        LogHelper.info("Doesn't type any words into Username textbox but enter valid information into Password textbox" +
+                " and click Login button");
+        loginPage.login("", Constant.PASSWORD);
+
+        LogHelper.info("Get error message");
+        String actual = loginPage.getErrorMessage();
+
+        LogHelper.info("Verify that user can't login with blank Username textbox");
+        Assert.assertEquals(actual, errorMsg, "An error message is not display like expected result");
+
```
Comment (tanmle) [inline src/test/java/tests/LoginTest.java]: redundant.

File: src/test/java/tests/LoginTest.java [hunk 2]
```diff
@@ -2,12 +2,14 @@
 
 import common.Constant;
 import helper.LogHelper;
+import org.slf4j.Logger;
```
Comment (MinhQuan119) [inline src/test/java/tests/LoginTest.java]: Remove un-used import

File: src/test/java/tests/LoginTest.java [hunk 3]
```diff
@@ -23,4 +25,21 @@ public void tc001_UserCanLogIntoRailwayWithValidUsernameAndPassword() {
         LogHelper.info("Verify that user can log into Railway with valid username and password");
         Assert.assertEquals(actual, "Welcome to Safe Railway", "Welcome message is not display like expected result");
     }
+
+    @Test(description = "User can't login with blank Username textbox")
+    public void tc02_UserCantLoginWithBlankUsernameTextbox() {
+        LogHelper.info("Click tab login");
+        loginPage.clickLoginTab();
+
+        LogHelper.info("Doesn't type any words into Username textbox but enter valid information into Password textbox" +
```
Comment (MinhQuan119) [inline src/test/java/tests/LoginTest.java]: You can make it simple by: "Login with empty username"

File: src/test/java/tests/LoginTest.java (hunk 2 above)
Comment (taile47711) [reply src/test/java/tests/LoginTest.java]: fixed

File: src/test/java/tests/LoginTest.java (hunk 3 above)
Comment (taile47711) [reply src/test/java/tests/LoginTest.java]: fixed

File: src/test/java/tests/LoginTest.java (hunk 1 above)
Comment (MinhQuan119) [reply src/test/java/tests/LoginTest.java]: @taile47711 you are missing this comment

Comment (taile47711) [reply src/test/java/tests/LoginTest.java]: fixed

**Review by MinhQuan119 — APPROVED** [review]

**Review by tanmle — APPROVED** [review]

**Pull request merged**